src/docfx/config/Config.cs

```diff
@@ -3,22 +3,33 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Docs.Build
 {
     internal class Config
     {
+        private static readonly string[] s_defaultContentInclude = new[] { "docs/**/*.{md,yml,json}" };
+        private static readonly string[] s_defaultContentExclude = Array.Empty<string>();
+
+        private static class Constants
```

Comment:
why not just put the strings to Config class

Reply:
already have a key in this class. will create a ConfigConstants class.

---
In reply to: [185455470](https://github.com/dotnet/docfx/pull/2702#discussion_r185455470) [](ancestors = 185455470)